src/Manager/DefaultManager.php

```diff
@@ -190,7 +190,7 @@ private function createTag(string $value): Tag
      *
      * @return array
      */
-    private function getCriteria(array $criteria = []): array
+    protected function getCriteria(array $criteria = []): array
```

Comment:
What is the reason you changed the properties and methods visibility?

Reply:
We adjusted it so that inheriting services can use the attributes and methods themselves without needing to redefine them.